templates/ContentGenerator/Instructor/UserDetail/set_row.html.ep

```diff
@@ -2,29 +2,40 @@
 %
 % # my ($set, $userSet, $mergedSet, $version) = @_;
 % my $setID   = $set->set_id;
+% my $isGateway = $set->assignment_type =~ /gateway/;
 % my $version = stash 'version';
 <tr>
-	<td class="text-center">
-		<label class="form-check-label">
-			<%= check_box $version ? "set.$setID,v$version.assignment" : "set.$setID.assignment" => 'assigned',
-				class => 'form-check-input', defined $mergedSet ? (checked => undef) : () =%>
-		</label>
-	</td>
-	<td class="text-center">
+	<th class="text-center" scope="row">
 		% if (defined $mergedSet) {
-			<b dir="ltr">
+			<span dir="ltr" class="px-2">
```

Comment:
I just noticed this `px-2` class.  What were you trying to do with that?  This can cause wrapping to occur before it needs to, and is inconsistent with the unassigned set case below.

Reply:
It also doesn't seem to do much that is useful since the text is centered in this table cell.

Reply:
I'm not sure. I think that was from yesterday. I would guess that for a set with a wide name, it looked like it got too close to the edge. I'll remove that and the `textarea` and push it back up.

Reply:
I though that might be the case.  For that it would be better to add padding to the table cell instead of padding to the text in the cell.